doc/tech/README.md

```diff
@@ -3,203 +3,170 @@
 
-<p align="center"><img src="./figures/tech_virtio.png" alt="virtio" title="Document Types" width="50%"/></p>
-<p align="center"><b>Figure 4:</b> Para-Virtualized interface components (software only).</p>
+  - Initially should be based on widely established technology and systems used in the Telecom Industry.  This will help ensure a faster adoption rate because the operators are already familiar with the technology and might even have systems in production. Another advantage to this approach is a project faster development cycle.
+  - Subsequent architectures should be based on either additional established or promising emerging technologies that are chosen by the community members.  
 
-<a name="4.1.2"></a>
-#### 4.1.2 Direct assignment with IOMMU.
+<a name="1.4"></a>
+## 1.4 Common NFVI Benefits
+By providing a pre-defined environment with common capabilities, applications are able to be developed and deployed more rapidly.  In addition, the common infrastructure can be optimized for various workloads, such as IT (Information Technology), VNF, AI (Artificial Intelligence), and other future workload types as new technologies emerge. The benefits of this approach are:
 
-Direct Assignment is supported in x86 architectures through an IOMMU (Input/Ouput Memory Management Unit), which provides the ability for a PCIe device to autonomously (i.e. without hypervisor intervention) perform DMA transfers directly into guest memory as shown in **Figure 5**.
+- Configuration automation over customization
+  - By abstracting the infrastructure capabilities as much as possible, operators are able to use common infrastructure platforms across all VNF vendors.
+  - Maintaining a consistent infrastructure allows for higher levels of automation due to a reduced need for customization of the various components.
+  - Overall, the intention is to reduce the total cost of ownership for operators and development costs for vendors
 
-Once an IO device is directly assigned to a workload, that workload will then have an exclusive access to that device and no other entities (including the hypervisor) can access it.
+- Onboarding and certification
+  - By defining abstracted infrastructure capabilities, and the metrics by which they are measured, the onboarding and certification process for both NFVI and VNFs can be standardized, reducing development time for the VNF developers and deployment and operational management costs for the operators standing up the NFVI environments.
+  - Supply chain, procurement and assurance teams can then use these metrics to more accurately assess the most efficient / best value vendor for a given environment and network services requirement.
 
-This method provides better performance than the para-virtualized one as no hypervisor is involved but provides less flexibility and less portability.
+- Better utilization
```

Comment:
utilization -> utilisation

Reply:
fixed in: https://github.com/cntt-n/CNTT/pull/1267/commits/ccd069304d5bb56d554fd8aaf2a8f02c8baa875e